OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -214,8 +214,12 @@ public AssetBrowserLogic(Widget widget, Action onExit, ModData modData, World wo
 				prevButton.IsVisible = () => !isVideoLoaded;
 			}
 
-			if (logicArgs.ContainsKey("SupportedFormats"))
-				allowedExtensions = FieldLoader.GetValue<string[]>("SupportedFormats", logicArgs["SupportedFormats"].Value);
+			var hasssetBrowserFileEndingsFilter = modData.Manifest.Contains<AssetBrowserFileEndingsFilter>();
```

Comment:
This check isn't needed - `Manifest.Get` will create an instance of the module with default values if it hasn't been explicitly defined in `mod.yaml`.

If you change the default value for `AssetBrowserFileEndingsFilter.SupportedFormats` to an empty array then you should be able to do just
```csharp
allowedExtensions = modData.Manifest.Get<AssetBrowserFileEndingsFilter>().SupportedFormats;
```
here.